packages/deployed-backend-client/src/deployed_backend_client_factory.ts

```diff
@@ -95,10 +96,14 @@ export enum BackendDeploymentStatus {
   UNKNOWN = 'UNKNOWN',
 }
 
+export enum BackendStatusFilter {
```

Comment:
```suggestion
export enum BackendStatus {
```
It's not a filter but a status, somebody filters with these statuses